app/controllers/admin/application_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::ApplicationController < ApplicationController
+  include AuthHelper
+  before_action :authenticate_user!, :authorize
+  helper_method :current_user
```

Comment:
it should be included by AuthHelper w/o extra declaration

Reply:
AuthHelper knows nothing about controller context, it could not be there